app/main/index.html

```diff
@@ -1,44 +0,0 @@
-<DOCTYPE html>
```

Comment:
In general, we should avoid creating additional index.html files like this until we get past the Sept Exec Demo